PokeAlarm/Manager.py

```diff
@@ -847,6 +847,14 @@ def process_egg(self, egg):
             return
 
         gym_id = egg['id']
+        gym_info = self.__cache.get_gym_info(gym_id)
+        gym_name = gym_info['name'].lower()
```

Comment:
Instead of making this a variable, let's just use the actual `gym_info['name'].lower()` in the contains check. This will prevent anyone from accidentally using it and getting a name in lower case.